qa/suites/upgrade/client-upgrade-kraken/kraken-client-x/rbd/1-install/kraken-client-x.yaml

```diff
@@ -4,7 +4,7 @@ tasks:
     exclude_packages: ['ceph-mgr','libcephfs2','libcephfs-devel','libcephfs-dev']
 - print: "**** done install jewel"
 - install.upgrade:
-   exclude_packages: ['ceph-test', 'ceph-test-dbg','libcephfs1']
+   exclude_packages: ['ceph-base', 'ceph-mon', 'ceph-osd', 'ceph-test', 'ceph-test-dbg','libcephfs1']
    client.1:
 - print: "**** done install.upgrade to -x on client.0"
```

Comment:
should we do s/client.0/client.1/ here?

Reply:
judging from the yum command [1] in the log, the packages list does not contain ceph-{base,mon,osd} explicitly, but rather just "ceph". So I wonder if this exclude will work.

[1] https://tracker.ceph.com/issues/22558#note-11

Reply:
it's not in the scope of this change. but if you insist, i will add another commit in this PR.

Reply:
I could open another PR to fix the documentation strings, if you insist :-P

Reply:
@smithfarm agreed, it won't work. 

- the ceph rpm package will bring in ceph-{osd,mds,mgr,mon} on , and
- the ceph deb package will bring in ceph-{mgr,mon,osd}.

i will also list "ceph" in the exclude list.

Reply:
whoa, I just noticed: `sudo yum -y install '' libcephfs_jni1 rbd-fuse ceph-radosgw librbd1 ceph-fuse python-ceph f ceph-mgr ceph librados2 cephfs-java libcephfs2 libcephfs-devel`

There appears to be a package `f` specified. . . that's weird.